widgets/BGDialog.cpp

```diff
@@ -234,5 +235,9 @@ void BGDialog::changeZ(int value)
 void BGDialog::updateBG()
 {
 	if(!field)	return;
-	image->setPixmap(QPixmap::fromImage(field->background()->openBackground(params, z, layers)));
+	image->setPixmap(QPixmap::fromImage(field->background()->openBackground(params, z, layers)).scaled(image->width(),image->height(),Qt::KeepAspectRatio,Qt::SmoothTransformation));

```

Comment:
Do you really need to scale the background? I need to check if SmoothTransformation is the best option.